client/i18n/locales/english/translations.json

```
@@ -716,5 +716,12 @@
     "focus-instructions-panel": "Focus Instructions Panel",
     "navigate-previous": "Navigate Previous Exercise",
     "navigate-next": "Navigate Next Exercise"
+  },
+  "signout": {
+    "heading": "Sign out of your account",
+    "p1": "Warning: Your progress will not be saved if you proceed.",
```

```suggestion
    "p1": "Warning: Your progress will stop being saved if you proceed.",
```

Maybe a little clearer? Don't want people to assume/misinterpret that signing out will remove any progress.

---

```suggestion
    "p1": "Warning: Signing out will stop saving your progress.",
```

You are correct. How about this — a more direct tone? If you agree, I will change the second paragraph slightly too (to be less repetitive).

---

Perfect.